sycl/test/usm/smem_varied.cpp

```diff
@@ -1,4 +1,5 @@
 // XFAIL: cuda
+// UNSUPPORTED: linux
```

Comment:
Why? According to your comments only `sycl/test/usm/smemll.cpp` hangs with new GPU driver. Right?